docs/usage.rst

```diff
@@ -0,0 +1,249 @@
+can be customized.
+
+Report Class
+~~~~~~~~~~~~
+
+The :py:class:`~edvart.report.Report` class is central to the edvart API.
+A *Report* consists of sections, which can be added via methods of the :py:class:`~edvart.report.Report` class.
+The class :py:class:`~edvart.report.DefaultReport` is a subclass of :py:class:`~edvart.report.Report`,
+which includes a default set of sections.
+
+With an instance of :py:class:`~edvart.report.Report` you can:
+
+1. Show the report directly in a Jupyter notebook using the :py:meth:`~edvart.report.Report.show` method.
+2. Export the code which generates the report to a new Jupyter notebook using
+   :py:meth:`~edvart.report.ReportBase.export_notebook` method.
+   The code can be exported with different levels of :ref:`verbosity <verbosity>`.
+   The notebook containing the exported code can be modified to fine-tune the report.
+3. Export the output to a HTML file. You can specify an
+   `nbconvert template
+   <https://nbconvert.readthedocs.io/en/latest/customizing.html#selecting-a-template>`_
+   to style the report.
+
+
+Selection of Sections
+~~~~~~~~~~~~~~~~~~~~~
+You can add sections using methods ``add_*`` (e.g. :py:meth:`edvart.report.ReportBase.add_overview`) of the :py:class:`~edvart.report.Report` class.
+
+.. code-block:: python
+
+    # Include univariate and bivariate analysis
+    import edvart
+    df = edvart.example_datasets.dataset_titanic()
+    report = (
+      edvart.Report(df)
+        .add_univariate_analysis()
+        .add_bivariate_analysis()
+    )
+
+.. _sections-config:
+
+Configuration of Sections
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each section can be also configured.
+For example you can define which columns should be used or omitted.
+
+.. code-block:: python
+
+  import edvart
+
+  df = edvart.example_datasets.dataset_titanic()
+  report = edvart.Report(df)
+
+  report.add_overview(omit_columns=["PassengerId"]).add_univariate_analysis(
+    use_columns=["Name", "Sex", "Age"]
+  )
+
+Subsections
+***********
+
+Some sections are made of subsections. For those, you can can configure which subsections are be included.
+
+.. code-block:: python
+
+    import edvart
+    from edvart.report_sections.dataset_overview import Overview
+
+    df = edvart.example_datasets.dataset_titanic()
+    report = edvart.Report(df)
+
+    report.add_overview(subsections=[
+        Overview.OverviewSubsection.QuickInfo,
+        Overview.OverviewSubsection.DataPreview,
+    ])
+
+
+.. _verbosity:
+
+Verbosity
+~~~~~~~~~
+
+A :py:class:`~edvart.report.Report` can be exported to a Jupyter notebook containing
+the code which generates the report. The code can be exported with different levels of detail,
+referred to as *verbosity*.
+
+It can be set on the level of the whole report or on the level of each
+section or subsection separately (see :ref:`sections-config`).
+
+The verbosity set on a lower level overrides the verbosity set on a higher level, i.e.
```

Comment:
This could be misleading. It can be understood such that `Verbosity.LOW` overrides `Verbosity.HIGH`. I'd word it as `Specific verbosity overrides general verbosity, i.e. ...`

Reply:
good point, changed